test/test_cpp20/custom_error.cpp

```diff
@@ -61,13 +61,10 @@ TEST_CASE("custom_error_logger")
     REQUIRE(fileNameSv.find("custom_error.cpp") != std::string::npos);
     const auto functionNameSv = std::string_view(s_loggerArgs.functionName);
     REQUIRE(!functionNameSv.empty());
-#if defined(__GNUC__) && !defined(__clang__)
-    REQUIRE(functionNameSv == "void {anonymous}::FailOnLine15()");
-#elif defined(__GNUC__) && defined(__clang__)
-    REQUIRE(functionNameSv == "void (anonymous namespace)::FailOnLine15()");
-#else
-    REQUIRE(functionNameSv == "FailOnLine15");
-#endif
+    // Every compiler has a slightly different naming approach for this function, and even the same
+    // compiler can change its mind over time.  Instead of matching the entire function name just
+    // match against the part we care about.
+    REQUIRE(functionNameSv.find("FailOnLine15" != std::string_view::npos));
```

Comment:
```suggestion
    REQUIRE((functionNameSv.find("FailOnLine15" != std::string_view::npos)));
```

Might need something like this to keep the preprocessor from exploding.

Reply:
Also, that looks like a typo: `find("FailOnLine15"`

Reply:
Wow, not sure how that compiled locally 🤦.  Fixed locally and will push as soon as build_test_all.cmd completes.

Reply:
The preprocessor works in mysterious ways. 😊